PocketCaddy/PocketCaddy/MainView/LaunchScene.swift

```diff
@@ -0,0 +1,50 @@
+
+import SpriteKit
+import GameKit
+extension UIImage {
+    static func gradientImage(withBounds: CGRect, startPoint: CGPoint, endPoint: CGPoint , colors: [CGColor]) -> UIImage {
+        
+        // Configure the gradient layer based on input
+        let gradientLayer = CAGradientLayer()
+        gradientLayer.frame = withBounds
+        gradientLayer.colors = colors
+        gradientLayer.startPoint = startPoint
+        gradientLayer.endPoint = endPoint
+        // Render the image using the gradient layer
+        UIGraphicsBeginImageContext(gradientLayer.bounds.size)
+        gradientLayer.render(in: UIGraphicsGetCurrentContext()!)
+        let image = UIGraphicsGetImageFromCurrentImageContext()
+        UIGraphicsEndImageContext()
+        
+        return image!
+    }
+}
+class GameScene: SKScene, SKPhysicsContactDelegate {
+    
+    var background = SKSpriteNode() // 백그라운드 노드
+    var flag = SKSpriteNode()    // 새 노드
```

Comment:
```suggestion
    var flag = SKSpriteNode()    // 새 노드

```

property와 method 사이에는 줄바꿈 하나로 구분되는게 좋습니다 ~~ !

Reply:
수정하였읍니다.